templates/workloads/services/lb-web/cf.yml

```diff
@@ -71,6 +71,10 @@ Resources:
 {{- if .Storage -}}
 {{include "mount-points" . | indent 10}}
 {{- end -}}
+{{- if .DockerLabels}}
```

Comment:
Seems like we have enough common fields for all workload types. What do you think of moving them to a `containerdef-base-properties` file?

Reply:
yeah, sounds good!